### ISSUE TITLE
Add support for application_name in pg_stat_activity

### ISSUE THREAD

**Issue by mateuszkj** [jasync-sql/jasync-sql#79]
It's very useful information when you have many client applications.

In JDBC it was:
```java
connection.setClientInfo("ApplicationName", "kaka-app")
```
or
```sql
SET application_name = 'kaka-app';
```

**Comment by andy-yx-chen** [comment]
Sure, why not

**Comment by oshai** [comment]
@andy-yx-chen are you going to contribute that?

**Comment by andy-yx-chen** [comment]
@oshai , sure, I will take a look at it

**Comment by andy-yx-chen** [comment]
tool an initial look, it could be as easy as execute a query after you get the connection, however, I am thinking to make this as a configuration for connection and support it for both mysql and pgsql, will find sometime to get  it done

**Comment by andy-yx-chen** [comment]
@oshai sorry for being late, but it's here https://github.com/jasync-sql/jasync-sql/pull/87

**Comment by andy-yx-chen** [comment]
this can be closed now

**Comment by oshai** [comment]
Fixed in 0.8.68
